shopinvader_schema_invoice/schemas/amount.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+from odoo.tools.float_utils import float_round
+
+
+class InvoiceAmount(StrictExtendableBaseModel, extra="ignore"):
```

Comment:
Why do you need to declare `extra="ignore"`? A common use of this parameter is for models used as incoming message. In this case we want to ignore unknown attributes.